.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,36 @@
+name: NPM publish
+
+on:
+  release:
+    types: [created]
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Package version'
+        required: true
+        default: v2.0.0
```

Comment:
Does NPM expect a `v` prefix as well?

Reply:
Not really, it always outputs version tags with the "v" prefix whether you supply that or not.
`npm version 0.0.2` -> tag = `v0.0.2` -> commit = `0.0.2`
`npm version v0.0.2` -> tag = `v0.0.2` -> commit = `0.0.2`
`npm version patch` -> tag = `v0.0.3` -> commit = `0.0.3` (if the previous version was 0.0.2)

So having it available in the default doesn't hurt I guess.

Reply:
👌